src/test/shared/filesystemUtilities.test.ts

```diff
@@ -49,6 +52,9 @@ describe('filesystemUtilities', () => {
 
         it('makes nested temp dirs', async () => {
             const nestedTempDirPath = await makeTemporaryToolkitFolder('nestedSubfolder', 'moreNestedSubfolder')
+
+            foldersToCleanUp.push(nestedTempDirPath)
```

Comment:
Will anything bad happen if the root folder for `makeTemporaryToolkitFolder` is deleted before this path?

Reply:
Nope